apps/native-component-list/navigation/RootNavigation.js

```diff
@@ -1,5 +1,7 @@
-import { StyleSheet } from 'react-native';
- import { createAppContainer } from 'react-navigation';
+import { createAppContainer } from 'react-navigation';
```

Comment:
Add a separate RootNavigation.web.js module -- we don't want to import non-web dependencies on web and vice versa.